server/src/main/java/com/linecorp/centraldogma/server/internal/api/auth/ApplicationTokenAuthorizer.java

```diff
@@ -57,6 +59,11 @@ public ApplicationTokenAuthorizer(Function<String, CompletionStage<Token>> token
     @Override
     public CompletionStage<Boolean> authorize(ServiceRequestContext ctx, HttpRequest data) {
         final OAuth2Token token = AuthTokenExtractors.oAuth2().apply(data.headers());
+        if (token != null && token.accessToken().equals(ApplicationToken.ANONYMOUS)) {
```

Comment:
@minwoox, thank you so much for the detailed explanation. It helped me a lot 👍 

It seems like `com.linecorp.centraldogma.client.armeria.AbstractCentralDogmaBuilder` sets `com.linecorp.centraldogma.internal.CsrfToken#ANONYMOUS` as accessToken by default. So the question is, should we use `CsrfToken#ANONYMOUS` when checking anonymous token, or should we create a new `ApplicationToken#ANONYMOUS` for the purpose of checking anonymous token?